IDEAS/BoundaryConditions/Climate/Meteo/Solar/RadSolData.mo

```diff
@@ -37,23 +27,34 @@ model RadSolData "Selects or generates correct solar data for this surface"
   Modelica.Blocks.Interfaces.RealOutput Tenv "Environment temperature"
     annotation (Placement(transformation(extent={{96,-30},{116,-10}})));
 protected
-      parameter Boolean isRoof = ceilingInc == inc
-    "Surface is a horizontal surface";
+  final parameter Boolean solDataInBus=if sum( { if sum(abs(incAndAziInBus[i,:] - {inc,azi}))<0.05 then 1 else 0 for i in 1:numIncAndAziInBus})   ==1 then true else false
```

Comment:
the outer if-else can be removed since  (if a then true else false) is the same as typing (a)